akka-management/core/src/main/scala/com/lightbend/lagom/internal/akka/management/AkkaManagementTrigger.scala

```diff
@@ -26,17 +27,22 @@ private[lagom] class AkkaManagementTrigger(
   coordinatedShutdown: CoordinatedShutdown
 )(implicit executionContext: ExecutionContext) {
 
-  private val enabled = config.getBoolean("lagom.akka.management.enabled")
+  private val enabledSetting = "lagom.akka.management.enabled"
+  private val enabledRenderedValue = config.getValue(enabledSetting).render()
```

Comment:
I'm adding this to avoid confusion for the user.

If a user uses: `on/off`, `yes/no` or `true/false`, the logging will display the value as defined by the user.